health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/CampaignsInbox.js

```
@@ -0,0 +1,26 @@
+  const { pathname } = useLocation();
+  const pastCampaignsInbox = pathname.includes("past-campaigns");
+  const config = pastCampaignsInbox
+    ? CampaignsInboxConfig({ headerLabel: t("PAST_CAMPAIGNS") })?.CampaignsInboxConfig?.[1]
+    : CampaignsInboxConfig({ headerLabel: t("LIVE_CAMPAIGNS") })?.CampaignsInboxConfig?.[0];
+  if (!config) {
+    return <Loader />;
+  }
+  return (
+    <React.Fragment>
+      <div className="digit-inbox-search-wrapper">
+        <InboxSearchComposer configs={config}></InboxSearchComposer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)